latex/sycl-1.2.1.tex

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
Why?

Reply:
I don't remember doing this. It may be my editor playing tricks on me again. I'll revert this immediately.

Reply:
That is why it is useful to look at the differences before committing... :-)

Reply:
You are so absolutely right. Bad habits.